pgbouncer_exporter.go

```diff
@@ -32,6 +33,12 @@ import (
 
 const namespace = "pgbouncer"
 
+type Config struct {
+	ConnectionString string `json:"pgBouncer.connectionString"`
+	MetricsPath      string `json:"web.telemetry-path"`
+	PidFilePath      string `json:"pgBouncer.pid-file"`
```

Comment:
We typically use yaml config files in Prometheus. This is more flexible as it allows for things like comments.